server/implementation/src/test/java/io/smallrye/graphql/execution/error/ErrorExtensionProvidersTest.java

```diff
@@ -0,0 +1,28 @@
+package io.smallrye.graphql.execution.error;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.smallrye.graphql.api.ErrorExtensionProvider;
+
+public class ErrorExtensionProvidersTest {
+    private final ErrorExtensionProviders providers = new ErrorExtensionProviders();
+
+    @Test
+    void should_get_same_instance() {
+        List<ErrorExtensionProvider> result1 = providers.get().collect(Collectors.toList());
+        List<ErrorExtensionProvider> result2 = providers.get().collect(Collectors.toList());
+
+        Assertions.assertThat(result1.size()).isEqualTo(3);
+        assertEquals(result1.size(), result2.size());
+        assertTrue(result1.get(0) == result2.get(0));
+        assertTrue(result1.get(1) == result2.get(1));
+        assertTrue(result1.get(2) == result2.get(2));
```

Comment:
This would be more assertj-ish:

```suggestion
        assertThat(result1).hasSize(3);
        assertThat(result2).hasSize(result1.size());
        assertThat(result1.get(0)).isSameAs(result2.get(0));
        assertThat(result1.get(1)).isSameAs(result2.get(1));
        assertThat(result1.get(2)).isSameAs(result2.get(2));
```

Reply:
fixed.